spec.html

```diff
@@ -6411,8 +6411,7 @@ <h1>GetNewTarget ( )</h1>
       <h1>GetGlobalObject ( )</h1>
       <p>The abstract operation GetGlobalObject returns the global object used by the currently running execution context. GetGlobalObject performs the following steps:</p>
       <emu-alg>
-        1. Let _ctx_ be the running execution context.
-        1. Let _currentRealm_ be _ctx_'s Realm.
+        1. Let _currentRealm_ be the current Realm record.
```

Comment:
```suggestion
        1. Let _currentRealm_ be the current Realm Record.
```
(We always capitalize "Realm Record".)